config/jobs-chromeos.yaml

```
@@ -472,7 +472,10 @@ _anchors:
       tree:
         - chromiumos
       branch:
+        - 'chromiumos:chromeos-5.15'
+        - 'chromiumos:chromeos-6.1'
         - 'chromiumos:chromeos-6.6'
+        - 'chromiumos:chromeos-6.12'
```

```suggestion
      min_version:
        version: 5
        patchlevel: 15
```

As this basically covers all branches >= 5.15, using a `min_version` rule makes more sense to me.